### ISSUE TITLE
Turbo.visit cache support

### ISSUE THREAD

**Issue by pySilver** [hotwired/turbo#1090]
I'm unsure if that is the right place to ask, but does `Turbo.visit` API has cache support? I'm building multi-level navigation for my site where accessing sub-menu triggers turbo frame requests. The problem is that navigating the same navigation sections is not immediate as it requests my backend repeatedly. That results in poor UX where user can hardly navigate through menu.

Ideally, I'd love to preload my navigation so users will not have to wait for it once they need it.

**Comment by brunoprietog** [comment]
Yes, `Turbo.visit` caches the response. Maybe [data-turbo-preload](https://turbo.hotwired.dev/handbook/drive#preload-links-into-the-cache) can cover your case?
